src/components/Post/GovernanceSideBar/Referenda/VoteReferendum.tsx

```diff
@@ -193,6 +193,12 @@ const VoteReferendum = ({ className, referendumId, onAccountChange, lastVote, se
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [track_number, address]);
 
+	useEffect(() => {

```

Comment:
please don't implement this using useEffect, this is bad for performance, listen to onChange events instead.